caravel/forms.py

```diff
@@ -212,6 +212,14 @@ def __init__(self, viz):
                 'Y',
                 choices=self.choicify(datasource.column_names),
                 description="Columns to display"),
+            'druid_time_origin': SelectField(
+                'Origin',
+                choices=(
+                    ('now', 'now'),
+                    ('2016-04-10', 'even'),
```

Comment:
yeah it's a bit weird because `even` can be used for lining up any granularity. Actually I should find a year where jan 1st was a Sunday. It'd work to line things up whether your granularity is minute, hour, day, sunday, year, ... I should probably put that in the tooltip.